packages/meditrak-app-server/src/__tests__/utilities/CentralApiMock.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+export class CentralApiMock {
+  public async registerUserAccount(userFields: Record<string, unknown>) {
+    return { message: 'Successfully created user', id: userFields.id };
+  }
```

Comment:
I wonder if we should inject this into the mock as part of the test suite that needs it? I found it was a little hard to trace back where that expected value was coming from, and in general having more specific mocks close to the tests they're being used in makes sense to me.

Reply:
Hmmm yeah that's a good idea 💡 I had also felt the flow was a bit weird. I'll re-write things to allow injecting mocks as needed 👍